x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/MvExpandOperator.java

```diff
@@ -161,7 +161,7 @@ private Page sliceExpandedIntoPages() {
                     prev.releaseBlocks();
                     prev = null;
                 }
-            }, expandedBlock);
+            }, expandedBlock::decRef);
```

Comment:
I prefer using `close` here (i.e., just using `expandedBlock`) because we don't explicitly use `incrRef`.